sources/packages/web/Dockerfile

```diff
@@ -1,15 +1,32 @@
-FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 as builder
+FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 AS builder
 
+# Application Port.
+ENV PORT 3030
 
 COPY ./package*.json ./
 RUN npm ci
 COPY . ./
+
+# Replace ${PORT} variable in the template and save as default.conf.
+RUN sed 's/${PORT}/'"${PORT}"'/g' default.conf.template > default.conf
```

Comment:
Is it not the same as having the PORT hardcode directly at the file?
Just wondering what is the benefit between having it hardcoded in the file vs the like 4.
Either way it is just a question for my information, not a blocker.

Reply:
In Dockerfile.dev it uses 8080 but for Dockerfile it uses 3030. We can change that later. It's just that in some places it has 3030 as the default port. We can change our local to 3030 too.